From: Shengjing Zhu <zhsj@debian.org>
Date: Fri, 1 May 2020 02:38:43 +0800
Subject: skip tests that need user namespace

---
 pkg/port/builtin/builtin_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pkg/port/builtin/builtin_test.go b/pkg/port/builtin/builtin_test.go
index 10d9472..8a7f145 100644
--- a/pkg/port/builtin/builtin_test.go
+++ b/pkg/port/builtin/builtin_test.go
@@ -16,6 +16,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestBuiltIn(t *testing.T) {
+	t.Skip("Need user namespace permission")
 	tmpDir, err := os.MkdirTemp("", "test-builtin")
 	if err != nil {
 		t.Fatal(err)
